web/libs/datamanager/src/components/App/App.scss

```diff
@@ -1,21 +1,3 @@
-body {
-  --black-raw: 0, 0, 0;
-  --white-raw: 255, 255, 255;
-  --accent_color-raw: 86, 111, 207;
-  --danger_color-raw: 221, 0, 0;
-  --black: rgb(var(--black-raw));
-  --white: rgb(var(--white-raw));
-  --accent_color: rgb(var(--accent_color-raw));
-  --danger_color: rgb(var(--danger_color-raw));
-
-  --black_2:  rgba(var(--black-raw), 0.02);
-  --black_4:  rgba(var(--black-raw), 0.04);
-  --black_5:  rgba(var(--black-raw), 0.05);
-  --black_10: rgba(var(--black-raw), 0.1);
-  --black_15: rgba(var(--black-raw), 0.15);
-  --black_20: rgba(var(--black-raw), 0.20);
-  --black_40: rgba(var(--black-raw), 0.40);
```

Comment:
looks like these were removed but not added to web/apps/labelstudio/src/themes/default/colors.scss

Reply:
i don't believe we need these anymore, i removed all instances that i could find - am i missing some?